### ISSUE TITLE
Empire VT-49 Decimator icon is incorrect

### ISSUE THREAD

**Issue by molotov681** [raithos/xwing#475]
Empire VT-49 Decimator icon is incorrect, shows a Tie Defender icon in ship selection drop down.

### Language

English

**Comment by Patschke** [comment]
I can't reproduce this - for me the Deci shows the Icon it is supposed to show. 
Is the Icon correctly displayed at https://geordanr.github.io/xwing-miniatures-font/ for you?

**Comment by raithos** [comment]
Are you using microsoft edge?

**Comment by molotov681** [comment]
It does show correctly on that page you linked me to. I am not using
Microsoft edge, I am using chrome. The use case was that I set the random
time to 50 seconds and set the points to 2000 and randomized and I believe
this caused the problem; I saw something similar on the rebel alliance page
as well.

**Comment by raithos** [comment]
thats... a really crazy use case.

**Comment by Patschke** [comment]
Well - this is clearly not a usecase the random generator is good at (as it will e.g. fill up all upgrade slots of the first few ships over time for sure), but nevertheless it works for me. I did not get any wrong icons, so I'm still unable to fix the bug you reported. 

Just as a note: I am not going to optimize the random generator to better handle 2k point squads ;-)

In case you wondered: [this was the squad I got](https://raithos.github.io/?f=Galactic%20Empire&d=v8ZsZ2000Z171X199W113W101Y191X114W134W102W98W88W70W71W110Y171X250W112W102Y172X132W240W102Y190X137W98W102W76W105Y214X122W234W33W37W49W90W70W165W147Y183X131W54W32W236Y181X126W105W109Y201X118W113W86W247Y199X124W114W82W165Y190X138W100W99W90W165Y209X119W88W68W166Y201X128W114W86W247Y170X75W111W102Y206X136W101W98W86W71W68W109Y166X111W12W32W30W236W162Y193X113W234W101W99W76W71W235WY182X166W106Y167XW13WW14W166WY164X113W136W98W104W139W134W102Y180X133W249W109Y163X115W113W134W102W166W144WY185X54WW249Y184X133W61W54W110Y216X134W30WW61W76W70W108WY201X115W114WWY223XWY161XWW134WWW144WY213XW68WY212XWW68WY187XWWWWWY208XWWWWWW&sn=Random%20Squad&obs=)
